apache/config/vhosts/standard.sls

```diff
@@ -5,6 +5,10 @@
 {%- set sls_package_install = tplroot ~ '.package.install' %}
 {%- set sls_service_running = tplroot ~ '.service.running' %}
 {%- from tplroot ~ "/map.jinja" import apache with context %}
+{#- The apache variable can grow _very_ large, especially the sites subkey.
+    Create a trimmed copy with config variables. #}
+{%- set map = apache %}
+{%- do map.pop('sites') %}
```

Comment:
Does Jinja2 support pop()?  I cannot find it at https://jinja.palletsprojects.com/en/2.11.x/

Reply:
It's not jinja. We're executing a direct python call on the object. https://docs.python.org/3/library/stdtypes.html#typesmapping

But I just added a small improvement that would guarantee we're not seeing any key not found errors. Thanks for reminding me of that.